src/modules/flickr.js

```diff
@@ -162,15 +163,44 @@
 			for (var i = 0; i < o.data.length; i++) {
 				var photo = o.data[i];
 				photo.name = photo.title;
-				photo.picture = getPhoto(photo.id, photo.farm, photo.server, photo.secret, '');
-				photo.source = getPhoto(photo.id, photo.farm, photo.server, photo.secret, 'b');
-				photo.thumbnail = getPhoto(photo.id, photo.farm, photo.server, photo.secret, 'm');
+				photo.picture = createPhotoUrl(photo.id, photo.farm, photo.server, photo.secret, '');
+				photo.pictures = createPictures(photo.id, photo.farm, photo.server, photo.secret);
+				photo.source = createPhotoUrl(photo.id, photo.farm, photo.server, photo.secret, 'b');
+				photo.thumbnail = createPhotoUrl(photo.id, photo.farm, photo.server, photo.secret, 'm');
 			}
 		}
 
 		return o;
 	}
 
+	// See: https://www.flickr.com/services/api/misc.urls.html
+	function createPictures(id, farm, server, secret) {
+
+		var NO_LIMIT = 2048;
+		var sizes = [
+			{id: 't', max: 100},
+			{id: 'm', max: 240},
+			{id: 'n', max: 320},
+			{id: '', max: 500},
+			{id: 'z', max: 640},
+			{id: 'c', max: 800},
+			{id: 'b', max: 1024},
+			{id: 'h', max: 1600},
+			{id: 'k', max: 2048},
+			{id: 'o', max: NO_LIMIT}
+		];
+
+		return sizes.map(function(size) {
+			return {
+				source: createPhotoUrl(id, farm, server, secret, size.id),
+
+				// Note: this is a guess that's almost certain to be wrong (unless square source)
+				width: size.max,
+				height: size.max
```

Comment:
Note: width and height doesn't come through the feed. We could work this out based on the rules outlined in https://www.flickr.com/services/api/misc.urls.html *if* we knew the aspect ratio, but this isn't present in the feed. It wouldn't be great if we didn't give any indicator of size, so this feels like it might be best option - but perhaps not!